web/pdf_find_controller.js

```diff
@@ -799,34 +820,22 @@ class PDFFindController {
         })
         .join("|");
     }
+    if (!query) {
+      return undefined;
+    }
```

Comment:
This seems to have lost the comment from https://github.com/mozilla/pdf.js/pull/18549/files#diff-693ef40ef030840f29fc58d0ec7dc7111114f096a6962d131aac3df4186e4430L677-L678